redfish-core/lib/pcie_slots.hpp

```diff
@@ -440,6 +441,33 @@ inline void
     size_t index = slots.size();
     slots.emplace_back(std::move(slot));
 
+    // Get and set the location code
+    sdbusplus::asio::getAllProperties(
```

Comment:
Please use get property when you are fetching just one property.

sdbusplus::asio::getProperty\<std::string\> (
        *crow::connections::systemBus, serviceName, fabricAdapterPath,
        "xyz.openbmc_project.Inventory.Decorator.LocationCode", "LocationCode",
        [aResp](const boost::system::error_code ec,
                const std::string& property){}